### PR TITLE
[SEM-016]: added molecule component side-panel 

### PR DESCRIPTION
**Pull request by rodrigoslucka**
Como usuario, quiero ver un componente lateral que liste diferentes tags, para poder acceder fácilmente a diferentes temas o categorías del sitio.

### CODE REVIEW

**Comment by DanX3N0** [comment]
figma: 
image
pr:
image
image

the variations primary and secondary are the same

File: src/ui/components/molecules/side-panel/mobile-side-pandel/MSPanel.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  component: MSPanel,
+  argTypes: {},
+};
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Main: Story = {
+  args: {
+    variant: 'main',
```
Comment (Ditmar) [inline src/ui/components/molecules/side-panel/mobile-side-pandel/MSPanel.stories.tsx:15]: I think will be better to use default like a variant

File: src/ui/components/molecules/side-panel/panel/SPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+function SPanel({ color = 'primary' }: IProps) {
+  const tags = ['workings', 'optimism', 'meaning', 'promoting'];
+  const getColorClassName = () => {
+    return color === 'primary' ? 'panel--primary' : 'panel--secondary';
+  };
+
+  return (
+    <div className={`panel ${getColorClassName()}`} data-testid="s-panel">
+      <LogoFooter icon={logo} color="primary" />
+      <Tags labels={tags} />
```
Comment (Ditmar) [inline src/ui/components/molecules/side-panel/panel/SPanel.tsx:17]: Hello, it is my fault for not checking the Tags well, this component should be current as simple links, we will use navigation by # to go through the articles.
I'm thinking of another approach, It is probably it was necessary to implement a pub-sub system to communicate with all entry applications, let me think a little bit, but for a while please modify this component

**Review by Ditmar — CHANGES_REQUESTED** [review]
Nice work please read these coments

**Review by Ditmar — APPROVED** [review]

**Review by Ditmar — CHANGES_REQUESTED** [review]
There are  some errors in the design  
mobile
image
Desktop 
image
In desktop mode the menu are so big

**Pull request merged**